scraper/src/github-scraper/discussion.ts

```diff
@@ -2,9 +2,7 @@ import { octokit } from "./config.js";
 import { Discussion, ParsedDiscussion } from "./types.js";
 import { saveDiscussionData } from "./utils.js";
 
-// Query to fetch discussions from GitHub
-const query = `
-query($org: String!, $cursor: String) {
+const query = `query($org: String!, $cursor: String) {
   organization(login: $org) {
     repositories(first: 100, after: $cursor) {
```

Comment:
repositories also needs to be sorted by updated at desc. right? So that we ensure that discussions are always scraped for even the oldest active projects.